src/guides/wallet-service/wallet-service.md

```diff
@@ -175,5 +175,70 @@ Added https options. Example:
 
 Notice that you can also specify which port your Wallet Service will run on (default is 3232).
 
+### Using free TLD and free SSL certificate to allow SSL/TLS support from mobile clients
+Self-signed certificates do not work very well, so using a TLD domain along with a root CA signed SSL certificate is the best method to ensure you can connect.
+
+First, get a free TLD from [http://www.dot.tk/](http://www.dot.tk/) and set the DNS to point to your server IP address. (Must be renewed every 12 months.)
```

Comment:
Perhaps we could assume a TLD has already been setup, e.g. through any domain registrar.